packages/docs/docs/dev/hooks-reference.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Hooks Reference
+category: For Developers
+---
+
+# React Hooks Reference
+
+## useAccount
+
+Retrieves the current fuel account and returns the account address `<string | undefined>`.
+
+```tsx
+const { account } = useAccount();
+console.log(account);
+// fuel1r20zhd...
+```
+
+[See the source file](https://github.com/FuelLabs/fuels-wallet/blob/master/packages/react/src/hooks/useAccount.tsx)
+
+## useBalance
+
+Fetches the balance `<number | undefined>` of a specified address and asset ID. Additionally, it includes a listener that triggers a balance refresh when the window gains focus.
+
+```tsx
+const { balance } = useBalance({
+  address: 'fuel1r20zhd...',
+  assetId: '0x000000000...',
+});
+
+console.log(balance);
+// 1000 (example balance)
+```
+
+[See the source file](https://github.com/FuelLabs/fuels-wallet/blob/master/packages/react/src/hooks/useBalance.tsx)
+
+## useChain
```

Comment:
We should show this in the networks page too.

Reply:
Thanks for the review!
I've added examples for networks, accounts, and signing;